task/02-numbers-tasks.js

```diff
@@ -202,7 +202,11 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+
+    for(let i = 2; i <= Math.sqrt(n); i++)
+        if(n % i === 0 )
+            return false
+    return true        
```

Comment:
don't forget to use curly braces, it improves the code readability
https://javascript.info/coding-style